docs/openapi.json

```diff
@@ -1742,16 +1742,20 @@
                 "type": "object",
                 "properties": {
                     "first": {
-                        "type": "string"
+                        "type": "string",
```

Comment:
This file is autogenerated from gocode annotations so we can not update it manually. But we can do it other way.

Reply:
yup. I'm trying to learn it :)